tools/Azure.Mcp.Tools.Extension/src/Options/ExtensionOptionDefinitions.cs

```diff
@@ -5,6 +5,27 @@ namespace Azure.Mcp.Tools.Extension.Options;
 
 public static class ExtensionOptionDefinitions
 {
+    public static class CliGenerate
+    {
+        public const string IntentName = "intent";
+        public static readonly Option<string> Intent = new(
+            $"--{IntentName}"
+        )
+        {
+            Description = "The user intent of the task to be solved by using the CLI tool. This user intent will be used to generate the appropriate CLI command to accomplish the desirable goal.",
+            Required = true
+        };
+
+        public const string CliTypeName = "cli-type";
```

Comment:
I think having 'cli- in the param name is redundant with the command name.  can it just be --type?